src/widget/form/genericchatform.cpp

```diff
@@ -199,6 +203,7 @@ GenericChatForm::GenericChatForm(QWidget *parent)
             this, &GenericChatForm::onChatContextMenuRequested);
 
     new QShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_L, this, SLOT(clearChatArea()));
+    new QShortcut(Qt::ALT + Qt::Key_Q, this, SLOT(quoteSelectedText()));
```

Comment:
Maybe the system should works differently. Example : copy a quote with ctrl+shift+c or r-click, this would simply add the quote in the clipboard. Letting the user do a simple ctrl+v, this way feels more intuitive/natural to me.